ui/cypress/integration/slow_query/list.spec.js

```diff
@@ -236,6 +241,363 @@ describe('SlowQuery list page', () => {
             })
         })
       })
+
+      it('Run workload without use database', () => {
+        let queryData = {
+          query: 'SELECT sleep(1.5);',
+          database: '',
+        }
+        cy.task('queryDB', { ...queryData })
+
+        cy.wait(2000)
+        cy.reload()
+        // global and use database queries will be listed
+        cy.get('[data-automation-key=query]').should('has.length', 3)
```

Comment:
What is `[data-automation-key=query]`? I only find `[data-automation-key=name]` in html structs.

Reply:
Each table column has a `data-automation-key` generated with `visibleColumnKeys` in CardTable.

![image](https://user-images.githubusercontent.com/34967660/154404151-1f97762d-1327-4573-8786-496d03aeb2e0.png)